slim/README.md

```diff
@@ -178,7 +178,7 @@ image classification dataset.
 In the table below, we list each model, the corresponding
 TensorFlow model file, the link to the model checkpoint, and the top 1 and top 5
 accuracy (on the imagenet test set).
-Note that the VGG and ResNet parameters have been converted from their original
+Note that the VGG and ResNet v1 parameters have been converted from their original
 caffe formats
 ([here](https://github.com/BVLC/caffe/wiki/Model-Zoo#models-used-by-the-vgg-team-in-ilsvrc-2014)
 and
```

Comment:
Can you change below 
whereas the Inception parameters have been trained internally at 
whereas the Inception and ResNet_v2 parameters have been trained internally at